### PR TITLE
refactor:  Input, TextArea, FormGroupLabel + Select to system props

### PR DESCRIPTION
**Pull request by dreamwasp**
### Overview
Updates Input, TextArea, FormGroupLabel + Select to system props (+ sets up the framework to move all other form fields over, which I will do in later PRs)

Monolith integration [here](https://github.com/codecademy-engineering/Codecademy/pull/24774)

### PR Checklist

- [X] Related to designs:
- [X] Related to JIRA ticket: [GM-215](https://codecademy.atlassian.net/browse/GM-215)
- [X] I have run this code to verify it works
- [X] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.12.3-alpha.6613b8.0<br/>@codecademy/gamut-illustrations@0.14.4-alpha.6613b8.0<br/>@codecademy/gamut-kit@0.3.101-alpha.6613b8.0<br/>@codecademy/gamut-labs@16.0.6-alpha.6613b8.0<br/>@codecademy/gamut-patterns@0.1.4-alpha.6613b8.0<br/>@codecademy/gamut-styles@10.0.1-alpha.6613b8.0<br/>@codecademy/gamut-tests@2.3.31-alpha.6613b8.0<br/>@codecademy/gamut@32.18.8-alpha.6613b8.0<br/>@codecademy/konamimojisplosion@0.2.31-alpha.6613b8.0<br/>@codecademy/markdown-overrides@0.5.43-alpha.6613b8.0<br/>@codecademy/styleguide@37.5.9-alpha.6613b8.0<br/>@codecademy/variance@0.11.3-alpha.6613b8.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60a2beef9a9e2d0095f56c39--gamut-preview.netlify.app>https://60a2beef9a9e2d0095f56c39--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60a2beef9a9e2d0095f56c39>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.114-alpha.eff7a6.0<br/>@codecademy/gamut-labs@18.1.5-alpha.eff7a6.0<br/>@codecademy/gamut@33.1.1-alpha.eff7a6.0<br/>@codecademy/styleguide@40.0.2-alpha.eff7a6.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60ad6b7f28462a07d9776d88--gamut-preview.netlify.app>https://60ad6b7f28462a07d9776d88--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60ad6b7f28462a07d9776d88>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.118-alpha.9b71fa.0<br/>@codecademy/gamut-labs@18.2.1-alpha.9b71fa.0<br/>@codecademy/gamut@33.1.4-alpha.9b71fa.0<br/>@codecademy/styleguide@40.0.6-alpha.9b71fa.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60ae561fcba0ad0093ba6c27--gamut-preview.netlify.app>https://60ae561fcba0ad0093ba6c27--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60ae561fcba0ad0093ba6c27>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.119-alpha.f8ea7e.0<br/>@codecademy/gamut-labs@18.2.2-alpha.f8ea7e.0<br/>@codecademy/gamut@33.1.5-alpha.f8ea7e.0<br/>@codecademy/styleguide@40.0.7-alpha.f8ea7e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60ae7b7152f6862f7d78aac0--gamut-preview.netlify.app>https://60ae7b7152f6862f7d78aac0--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60ae7b7152f6862f7d78aac0>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.14.3-alpha.55fdc6.0<br/>@codecademy/gamut-kit@0.3.119-alpha.55fdc6.0<br/>@codecademy/gamut-labs@18.2.2-alpha.55fdc6.0<br/>@codecademy/gamut@33.1.5-alpha.55fdc6.0<br/>@codecademy/styleguide@40.0.7-alpha.55fdc6.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60ae80118cd8a00097b83923--gamut-preview.netlify.app>https://60ae80118cd8a00097b83923--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60ae80118cd8a00097b83923>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.120-alpha.6242b7.0<br/>@codecademy/gamut-labs@18.2.3-alpha.6242b7.0<br/>@codecademy/gamut@33.1.6-alpha.6242b7.0<br/>@codecademy/styleguide@40.0.8-alpha.6242b7.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60ae836a5b0b0600c04477a7--gamut-preview.netlify.app>https://60ae836a5b0b0600c04477a7--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60ae836a5b0b0600c04477a7>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.120-alpha.261b4a.0<br/>@codecademy/gamut-labs@18.2.3-alpha.261b4a.0<br/>@codecademy/gamut@33.1.6-alpha.261b4a.0<br/>@codecademy/styleguide@40.0.8-alpha.261b4a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60ae8ab1354cd010d749de6c--gamut-preview.netlify.app>https://60ae8ab1354cd010d749de6c--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60ae8ab1354cd010d749de6c>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.120-alpha.916f15.0<br/>@codecademy/gamut-labs@18.2.3-alpha.916f15.0<br/>@codecademy/gamut@33.1.6-alpha.916f15.0<br/>@codecademy/styleguide@40.0.8-alpha.916f15.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60aec9dc49176f119e1bfb4e--gamut-preview.netlify.app>https://60aec9dc49176f119e1bfb4e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60aec9dc49176f119e1bfb4e>Deploy Logs</a></p>

File: packages/gamut/src/Form/styles/shared-system-props.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  transition: string
+) => {
+  const cssString = `${arrayOfProperties.join(
+    ` ${transition},`
+  )} ${transition}`;
+
+  return {
+    transition: cssString,
+  };
+};
```
Comment (codecaaron) [inline packages/gamut/src/Form/styles/shared-system-props.tsx:29]: Could we move this to gamut-styles and key it with our timings?

```ts
import { DataType } from 'csstype'; 

const transitionConcatenator = (
  arrayOfProperties: Array<keyof StandardPropertiesHyphen>,
  timing: keyof typeof timings,
  timingFn:  DataType.EasingFunction  = 'linear',
) => {
  const cssString = `${arrayOfProperties.join(
    ` ${transition},`
  )} ${transition}`;

  return {
    transition: cssString,
  };
};
```

Also might be nice to just return the string instead of the object so we can use it like:
```tsx
system.css({
  transition: transitionConcat(['background-color', 'color'], 'fast', 'ease-in-out');
})

File: packages/gamut/src/Form/TextArea.tsx [hunk 2]
```diff
@@ -18,7 +23,7 @@ export type TextWrapperProps = TextareaHTMLAttributes<HTMLTextAreaElement> & {
 };
 
 export interface TextAreaProps extends TextWrapperProps {
-  activated?: boolean;
+  variant?: variantStyleType;
 }
```
Comment (codecaaron) [inline packages/gamut/src/Form/TextArea.tsx:27]: There's a helper to do the typescript inference for you so we don't have to set this everywhere :D 

```tsx
import { StyleProps } from '@codecademy/variance';

export interface TextAreaProps extends TextWrapperProps, StyleProps<typeof conditionalStyles> {}
```

File: packages/gamut/src/Form/Input.tsx [hunk 3]
```diff
@@ -56,8 +57,14 @@ export interface InputWrapperProps extends InputProps {
 
 /**  We greatly prefer NOT to do this but ReactRecurly has some specific needs around focus-styles + padding that force us to export them seperately. If we ever stop using React-Recurly, this code will be 🔪.
  *tldr: Do not do this unless you have already talked to Web-Plat and have failed to find any alternate (and better) solutions. */
-export const reactRecurlyFormFieldFocusStyles = formFieldFocusStyles;
-export const reactRecurlyFormFieldPaddingStyles = formFieldPaddingStyles;
+
+export const reactRecurlyFormFieldFocusStyles = system.css({
+  ...formFieldFocusStyles,
+});
```
Comment (codecaaron) [inline packages/gamut/src/Form/Input.tsx:63]: No issues with immutability here :) 

```suggestion
export const reactRecurlyFormFieldFocusStyles = system.css(formFieldFocusStyles);
```

File: packages/gamut/src/Form/Input.tsx [hunk 4]
```diff
@@ -68,8 +75,8 @@ export const iFrameWrapper = styled.div<conditionalInputStyleProps>`
 const InputElement = styled.input<StyledInputProps>`
   ${formFieldStyles}
   ${conditionalStyles}
-  ${iconPadding}
   text-indent: 0;
+  padding-right: ${(props) => (props.icon ? `2.3rem` : `initial`)};
```
Comment (codecaaron) [inline packages/gamut/src/Form/Input.tsx:79]: Hmm are we still using `iconPadding` variant out there?

File: packages/gamut/src/Form/styles/shared-system-props.tsx [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+  variants: {
+    error: {
+      textColor: 'danger',
+      borderColor: 'danger',
+      '&:hover': {
+        borderColor: 'danger',
+      },
+      '&:focus': {
+        borderColor: 'danger',
+        boxShadow: `inset 0 0 0 1px ${theme.colors.danger}`,
```
Comment (codecaaron) [inline packages/gamut/src/Form/styles/shared-system-props.tsx:105]: Trix 
```suggestion
        boxShadow: `inset 0 0 0 1px currentColor`,
```

File: packages/gamut/src/Form/styles/shared-system-props.tsx [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+  },
+  '&:disabled': {
+    ...formFieldDisabledStyles,
+    fontStyle: 'italic',
+  },
+} as const;
+
+export const formBaseFieldStyles = system.css({
+  ...formBaseFieldStylesObject,
+});
```
Comment (codecaaron) [inline packages/gamut/src/Form/styles/shared-system-props.tsx:87]:
```suggestion
export const formBaseFieldStyles = system.css(formBaseFieldStylesObject);
```

File: packages/gamut/src/Form/styles/shared-system-props.tsx [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+
+export const formFieldFocusStyles = {
+  borderColor: 'primary',
+  boxShadow: `inset 0 0 0 1px ${theme.colors.primary}`,
+} as const;
+
+const formFieldDisabledStyles = {
+  bg: 'backgroundEmphasized',
+  borderColor: 'textMuted',
+  color: 'textMuted',
```
Comment (codecaaron) [inline packages/gamut/src/Form/styles/shared-system-props.tsx:53]: I really gotta fix the JSX types for this, not being able to use color is super annoying.

```suggestion
  textColor: 'textMuted',
```

File: packages/gamut/src/Form/styles/shared-system-props.tsx [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+  fontStyle: 'italic',
+  cursor: 'not-allowed',
+  '&:hover': {
+    borderColor: 'textMuted',
+  },
```
Comment (codecaaron) [inline packages/gamut/src/Form/styles/shared-system-props.tsx:58]: I wonder if we convert the border colors to only read from `currentColor` (may not work but it's possible I think?) we could avoid some of these cases.

File: packages/gamut/src/Form/styles/shared-system-props.tsx [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+
+  return {
+    transition: cssString,
+  };
+};
+
+export const formBaseStyles = {
+  fontWeight: 'base',
+  fontSize: 16,
+} as const;
```
Comment (codecaaron) [inline packages/gamut/src/Form/styles/shared-system-props.tsx:34]: I think we can just add these to `formBaseComponentStyles`?

**Review by codecaaron — COMMENTED** [review]
Leeeeeet's Go!!!!

Amazing!

File: packages/gamut/src/Form/Input.tsx (hunk 4 above)
Comment (dreamwasp) [reply packages/gamut/src/Form/Input.tsx:77]: nope, i removed it in lieu of this!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.14.5-alpha.d5b649.0<br/>@codecademy/gamut-illustrations@0.17.1-alpha.d5b649.0<br/>@codecademy/gamut-kit@0.4.3-alpha.d5b649.0<br/>@codecademy/gamut-labs@18.3.3-alpha.d5b649.0<br/>@codecademy/gamut-patterns@0.3.1-alpha.d5b649.0<br/>@codecademy/gamut-styles@10.2.1-alpha.d5b649.0<br/>@codecademy/gamut-tests@2.4.4-alpha.d5b649.0<br/>@codecademy/gamut@33.4.1-alpha.d5b649.0<br/>@codecademy/konamimojisplosion@0.2.35-alpha.d5b649.0<br/>@codecademy/markdown-overrides@0.5.47-alpha.d5b649.0<br/>@codecademy/styleguide@40.3.1-alpha.d5b649.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60b15d0eabc557263c65bb3c--gamut-preview.netlify.app>https://60b15d0eabc557263c65bb3c--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60b15d0eabc557263c65bb3c>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.14.5-alpha.329204.0<br/>@codecademy/gamut-illustrations@0.17.1-alpha.329204.0<br/>@codecademy/gamut-kit@0.4.3-alpha.329204.0<br/>@codecademy/gamut-labs@18.3.3-alpha.329204.0<br/>@codecademy/gamut-patterns@0.3.1-alpha.329204.0<br/>@codecademy/gamut-styles@10.2.1-alpha.329204.0<br/>@codecademy/gamut-tests@2.4.4-alpha.329204.0<br/>@codecademy/gamut@33.4.1-alpha.329204.0<br/>@codecademy/konamimojisplosion@0.2.35-alpha.329204.0<br/>@codecademy/markdown-overrides@0.5.47-alpha.329204.0<br/>@codecademy/styleguide@40.3.1-alpha.329204.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60b162e22c611621b80fb16f--gamut-preview.netlify.app>https://60b162e22c611621b80fb16f--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60b162e22c611621b80fb16f>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.14.5-alpha.57abeb.0<br/>@codecademy/gamut-illustrations@0.17.1-alpha.57abeb.0<br/>@codecademy/gamut-kit@0.4.3-alpha.57abeb.0<br/>@codecademy/gamut-labs@18.3.3-alpha.57abeb.0<br/>@codecademy/gamut-patterns@0.3.1-alpha.57abeb.0<br/>@codecademy/gamut-styles@10.2.1-alpha.57abeb.0<br/>@codecademy/gamut-tests@2.4.4-alpha.57abeb.0<br/>@codecademy/gamut@33.4.1-alpha.57abeb.0<br/>@codecademy/konamimojisplosion@0.2.35-alpha.57abeb.0<br/>@codecademy/markdown-overrides@0.5.47-alpha.57abeb.0<br/>@codecademy/styleguide@40.3.1-alpha.57abeb.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60b166f7efb68e31375bfc0e--gamut-preview.netlify.app>https://60b166f7efb68e31375bfc0e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60b166f7efb68e31375bfc0e>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.14.5-alpha.c0bb22.0<br/>@codecademy/gamut-illustrations@0.17.1-alpha.c0bb22.0<br/>@codecademy/gamut-kit@0.4.5-alpha.c0bb22.0<br/>@codecademy/gamut-labs@18.3.5-alpha.c0bb22.0<br/>@codecademy/gamut-patterns@0.3.1-alpha.c0bb22.0<br/>@codecademy/gamut-styles@10.2.1-alpha.c0bb22.0<br/>@codecademy/gamut-tests@2.4.4-alpha.c0bb22.0<br/>@codecademy/gamut@33.4.3-alpha.c0bb22.0<br/>@codecademy/konamimojisplosion@0.2.35-alpha.c0bb22.0<br/>@codecademy/markdown-overrides@0.5.47-alpha.c0bb22.0<br/>@codecademy/styleguide@40.3.3-alpha.c0bb22.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60b16882abc557271665bbef--gamut-preview.netlify.app>https://60b16882abc557271665bbef--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60b16882abc557271665bbef>Deploy Logs</a></p>

File: packages/gamut/src/Form/TextArea.tsx [hunk 10]
```diff
@@ -42,8 +47,7 @@ export const TextArea = forwardRef<HTMLTextAreaElement, TextWrapperProps>(
         id={id || rest.htmlFor}
         className={className}
         ref={ref}
-        error={error}
-        activated={activated}
+        variant={conditionalStyleState(Boolean(error), activated)}
```
Comment (codecaaron) [inline packages/gamut/src/Form/TextArea.tsx:50]: Unrelated - Wonder if the `touched` key on `formState` might be helpful for keeping track of these states :) [`Docs`](https://react-hook-form.com/v6/api#formState)

File: packages/gamut/src/Form/Input.tsx [hunk 11]
```diff
@@ -56,10 +57,16 @@ export interface InputWrapperProps extends InputProps {
...
-export const reactRecurlyFormFieldPaddingStyles = formFieldPaddingStyles;
 
-export const iFrameWrapper = styled.div<conditionalInputStyleProps>`
+export const reactRecurlyFormFieldFocusStyles = system.css(
+  formFieldFocusStyles
+);
+
+export const reactRecurlyFormFieldPaddingStyles = system.css({
+  ...formFieldPaddingStyles,
+});
```
Comment (codecaaron) [inline packages/gamut/src/Form/Input.tsx:67]: Nit:
```suggestion
export const reactRecurlyFormFieldPaddingStyles = system.css(formFieldPaddingStyles);
```

File: packages/gamut/src/Form/Input.tsx (hunk 4 above)
Comment (codecaaron) [inline packages/gamut/src/Form/Input.tsx:79]: This might be helpful down the line!  https://github.com/Codecademy/client-modules/pull/1791

```tsx
const InputElement = styled.input(
  system.states({
     icon: {
        pr: 48, // or whatever looks nice :)
     }
  })
)
/>
<InputElement icon />
```

File: packages/gamut/src/Form/styles/shared.tsx [hunk 12]
```diff
@@ -24,20 +26,6 @@ export const colorStates = {
...
-
-type iconPaddingProps = {
-  icon?: boolean;
-};
-
-export type conditionalInputStyleProps = conditionalStyleProps &
-  iconPaddingProps;
-
 // these are split for now because ReactRecurly demands separate styles for focus.
 export const formFieldFocusStyles = css`
```
Comment (codecaaron) [inline packages/gamut/src/Form/styles/shared.tsx:30]: Are these still being used by other components?

**Review by codecaaron — APPROVED** [review]
Wonderful :)

File: packages/gamut/src/Form/TextArea.tsx (hunk 10 above)
Comment (dreamwasp) [reply packages/gamut/src/Form/TextArea.tsx:50]: yeah! i'm planning to refactor these to use formState (+ useController) in the future

File: packages/gamut/src/Form/Input.tsx (hunk 4 above)
Comment (dreamwasp) [reply packages/gamut/src/Form/Input.tsx:77]: 🙏

File: packages/gamut/src/Form/styles/shared.tsx (hunk 12 above)
Comment (dreamwasp) [reply packages/gamut/src/Form/styles/shared.tsx:30]: the remainder of the shared file is stuff that is being used in SelectDropdown which i would like to swap over in it's own PR

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.14.5-alpha.ef572e.0<br/>@codecademy/gamut-illustrations@0.17.1-alpha.ef572e.0<br/>@codecademy/gamut-kit@0.4.5-alpha.ef572e.0<br/>@codecademy/gamut-labs@18.3.5-alpha.ef572e.0<br/>@codecademy/gamut-patterns@0.3.1-alpha.ef572e.0<br/>@codecademy/gamut-styles@10.2.1-alpha.ef572e.0<br/>@codecademy/gamut-tests@2.4.4-alpha.ef572e.0<br/>@codecademy/gamut@33.4.3-alpha.ef572e.0<br/>@codecademy/konamimojisplosion@0.2.35-alpha.ef572e.0<br/>@codecademy/markdown-overrides@0.5.47-alpha.ef572e.0<br/>@codecademy/styleguide@40.3.3-alpha.ef572e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60b64162f776afbb2844588c--gamut-preview.netlify.app>https://60b64162f776afbb2844588c--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60b64162f776afbb2844588c>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.14.5-alpha.51a8de.0<br/>@codecademy/gamut-illustrations@0.17.1-alpha.51a8de.0<br/>@codecademy/gamut-kit@0.4.5-alpha.51a8de.0<br/>@codecademy/gamut-labs@18.3.5-alpha.51a8de.0<br/>@codecademy/gamut-patterns@0.3.1-alpha.51a8de.0<br/>@codecademy/gamut-styles@10.2.1-alpha.51a8de.0<br/>@codecademy/gamut-tests@2.4.4-alpha.51a8de.0<br/>@codecademy/gamut@33.4.3-alpha.51a8de.0<br/>@codecademy/konamimojisplosion@0.2.35-alpha.51a8de.0<br/>@codecademy/markdown-overrides@0.5.47-alpha.51a8de.0<br/>@codecademy/styleguide@40.3.3-alpha.51a8de.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60b6475a37549593f2cc1b8d--gamut-preview.netlify.app>https://60b6475a37549593f2cc1b8d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60b6475a37549593f2cc1b8d>Deploy Logs</a></p>

File: packages/gamut/src/Form/styles/shared-system-props.tsx [hunk 13]
```diff
@@ -0,0 +1,91 @@
...
+
+const formFieldDisabledStyles = {
+  bg: 'backgroundEmphasized',
+  borderColor: 'currentColor',
+  textColor: 'textMuted',
+  fontStyle: 'italic',
+  cursor: 'not-allowed',
+  '&:hover': {
+    borderColor: 'currentColor',
+  },
```
Comment (codecaaron) [inline packages/gamut/src/Form/styles/shared-system-props.tsx:33]:
```suggestion
```

**Review by codecaaron — COMMENTED** [review]
This is the one! EDIT: NO IT IS NOT

**Comment by codecaaron** [comment]
Fix for the issue is here: https://github.com/Codecademy/client-modules/pull/1797 - you don't need to change anything!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.14.5-alpha.cdddd7.0<br/>@codecademy/gamut-illustrations@0.17.1-alpha.cdddd7.0<br/>@codecademy/gamut-kit@0.4.5-alpha.cdddd7.0<br/>@codecademy/gamut-labs@18.3.5-alpha.cdddd7.0<br/>@codecademy/gamut-patterns@0.3.1-alpha.cdddd7.0<br/>@codecademy/gamut-styles@10.2.1-alpha.cdddd7.0<br/>@codecademy/gamut-tests@2.4.4-alpha.cdddd7.0<br/>@codecademy/gamut@33.4.3-alpha.cdddd7.0<br/>@codecademy/konamimojisplosion@0.2.35-alpha.cdddd7.0<br/>@codecademy/markdown-overrides@0.5.47-alpha.cdddd7.0<br/>@codecademy/styleguide@40.3.3-alpha.cdddd7.0<br/>@codecademy/variance@0.12.1-alpha.cdddd7.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60b78d2df9b9f90eb008aff8--gamut-preview.netlify.app>https://60b78d2df9b9f90eb008aff8--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60b78d2df9b9f90eb008aff8>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.14.6-alpha.106693.0<br/>@codecademy/gamut-illustrations@0.17.2-alpha.106693.0<br/>@codecademy/gamut-kit@0.4.6-alpha.106693.0<br/>@codecademy/gamut-labs@18.3.6-alpha.106693.0<br/>@codecademy/gamut-patterns@0.3.2-alpha.106693.0<br/>@codecademy/gamut-styles@10.2.2-alpha.106693.0<br/>@codecademy/gamut-tests@2.4.5-alpha.106693.0<br/>@codecademy/gamut@33.4.4-alpha.106693.0<br/>@codecademy/konamimojisplosion@0.2.36-alpha.106693.0<br/>@codecademy/markdown-overrides@0.5.48-alpha.106693.0<br/>@codecademy/styleguide@40.3.4-alpha.106693.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60b794f10025fc333a82392d--gamut-preview.netlify.app>https://60b794f10025fc333a82392d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60b794f10025fc333a82392d>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.14.6-alpha.18c8b0.0<br/>@codecademy/gamut-illustrations@0.17.2-alpha.18c8b0.0<br/>@codecademy/gamut-kit@0.4.7-alpha.18c8b0.0<br/>@codecademy/gamut-labs@18.4.1-alpha.18c8b0.0<br/>@codecademy/gamut-patterns@0.3.2-alpha.18c8b0.0<br/>@codecademy/gamut-styles@10.2.2-alpha.18c8b0.0<br/>@codecademy/gamut-tests@2.4.5-alpha.18c8b0.0<br/>@codecademy/gamut@33.4.4-alpha.18c8b0.0<br/>@codecademy/konamimojisplosion@0.2.36-alpha.18c8b0.0<br/>@codecademy/markdown-overrides@0.5.48-alpha.18c8b0.0<br/>@codecademy/styleguide@40.3.5-alpha.18c8b0.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60b7edd884aadc36b134ce64--gamut-preview.netlify.app>https://60b7edd884aadc36b134ce64--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60b7edd884aadc36b134ce64>Deploy Logs</a></p>
